### PR TITLE
Simplify Ubuntu and Debian monitoring

### PR DESCRIPTION
**Pull request by ajabep**
Instead of searching libc for each version, it will index all those available.

### CODE REVIEW

File: get [hunk 1]
```diff
@@ -2,32 +2,11 @@
...
-    stretch
-    buster
-    bullseye
-)
-for version in ${debian_versions[@]}; do
-  get_current_debian $version i386 libc6
-  get_current_debian $version amd64 libc6
-  get_current_debian $version amd64 libc6-i386
-done
+get_all_ubuntu ubuntu-eglibc http://archive.ubuntu.com/ubuntu/pool/main/e/eglibc/
```
Comment (niklasb) [inline get:5]: rename to get_all_debian

File: get [hunk 2]
```diff
@@ -2,32 +2,11 @@
...
-get_all_ubuntu archive-old-glibc http://old-releases.ubuntu.com/ubuntu/pool/main/g/glibc/
-
-debian_versions=(
-    jessie
-    stretch
-    buster
-    bullseye
-)
-for version in ${debian_versions[@]}; do
-  get_current_debian $version i386 libc6
```
Comment (niklasb) [inline get:30]: remove get_current_debian & get_current_ubuntu if they are no longer used

**Comment by niklasb** [comment]
I'm pretty sure I thought about this when I wrote the initial version? did you verify that the archive contains all the old versions?

**Comment by ajabep** [comment]
> I'm pretty sure I thought about this when I wrote the initial version? did you verify that the archive contains all the old versions?

Yep, the archive contains old version both for Ubuntu and Debian

**Pull request merged**